client/my-sites/theme/style.scss

```diff
@@ -4,6 +4,13 @@ $theme-sheet-content-max-width: 1462px;
 $button-border: 4px;
 
 .is-section-theme {
+	// overwrite notices z-index so they are visible on top of the theme preview
+	.global-notices {
+		// This should be - z-index: z-index("root", ".is-section-theme .global-notices")
+		// but the @automattic/onboarding/styles/mixins import overwrites the z-index function on client/assets/stylesheets/shared/functions/_z-index.scss
+		z-index: 100201;
+	}
```

Comment:
I don't like this approach. An alternative I thought could be having a second styles file with just 

```
.is-section-theme {
	// overwrite notices z-index so they are visible on top of the theme preview
	.global-notices {
		z-index: z-index("root", ".is-section-theme .global-notices")
	}
}
```

That wouldn't collide with the `z-index` function from `"@automattic/onboarding/styles/mixins"`, but it'd easy to overlook that the component is working with 2 different stylesheets 😓

Wdyt?

Reply:
I think directly using the value here is fine.

Calypso is riddled with absolute z-index values; IMHO, it's a byproduct of how Calypso outgrew its original SPA scope when it became a monorepo.

The stacking context is already very unclear anyway. Adding an exception won't make things much worse.

(Just to be clear, I'm really grateful you went out of your way to find the best solution!)